resources/org/eclipsefdn/hugoWebsite/Dockerfile

```diff
@@ -2,15 +2,15 @@ ARG BUILDER_IMAGE_TAG=latest
 ARG NGINX_IMAGE_TAG=stable-alpine-for-hugo
 
 FROM eclipsefdn/hugo-node:${BUILDER_IMAGE_TAG} AS builder
-# See https://github.com/EclipseFdn/dockerfiles/tree/master/hugo-node 
+# See https://github.com/EclipseFdn/dockerfiles/tree/master/hugo-node
 # and https://github.com/EclipseFdn/dockerfiles/blob/master/build.sh
 # for hugo and node versions (note the latest in the build.sh file)
 
 WORKDIR /workdir
 COPY . /workdir/
 
-RUN npm --unsafe-perm ci \
-    && hugo --gc --minify
+RUN if [ -e yarn.lock ]; then yarn install --frozen-lockfile; elif [ -e package-lock.json ]; then npm --unsafe-perm ci; fi; \
```

Comment:
why not just an `else` rather than `elif [ -e package-lock.json ]`? This dockerfile always assumed npm ci to be executed by default.

Reply:
Wanted to support a use-case where we might not have anything to build.

Reply:
ok, hopefully it won't break anything :D

Reply:
npm --unsafe-perm ci will fail without a package-lock.json file.